apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -176,39 +212,22 @@ export class ApplicationService {
   ): Promise<ApplicationModel | null> {
     const application = await this.applicationModel.findOne({
       where: { id },
-      include: [
-        { model: StaffModel, as: 'staff' },
-        {
-          model: ApplicationEventModel,
-          as: 'applicationEvents',
-          separate: true,
-          where: {
-            eventType: {
-              [Op.in]: isEmployee
-                ? Object.values(ApplicationEventType)
-                : [ApplicationEventType.DATANEEDED],
-            },
-          },
-          order: [['created', 'DESC']],
-        },
-        {
-          model: ApplicationFileModel,
-          as: 'files',
-          separate: true,
-          order: [['created', 'DESC']],
-        },
-        {
-          model: AmountModel,
-          as: 'amount',
-          include: [{ model: DeductionFactorsModel, as: 'deductionFactors' }],
-          separate: true,
-          order: [['created', 'DESC']],
-        },
-        {
-          model: DirectTaxPaymentModel,
-          as: 'directTaxPayments',
-        },
-      ],
+      include: applicationInlcudeClause(isEmployee),
+    })
+
+    if (application?.amount) {
+      application.setDataValue('amount', application.amount['0'])
+    }
+
+    return application
+  }
+
+  async findByApplicationSystemId(
+    id: string,
+  ): Promise<ApplicationModel | null> {
+    const application = await this.applicationModel.findOne({
+      where: { applicationSystemId: id },
+      include: applicationInlcudeClause(false),
```

Comment:
why are you not just using the  findById endpoint, rather than making a new one?

Reply:
I did it because I thought we didn't have access to the application id. But I realised we can get the id from the response of the createApplication request.
So I have removed the unnecessary endpoint and reusing the old one.